services/api-server/src/simcore_service_api_server/models/schemas/meta.py

```diff
@@ -1,12 +1,16 @@
 from typing import Annotated
 
 from models_library.api_schemas__common.meta import BaseMeta
-from pydantic import AnyHttpUrl, ConfigDict, StringConstraints
+from models_library.basic_types import VersionStr
+from pydantic import AnyHttpUrl, ConfigDict, Field, StringConstraints
 
 
 class Meta(BaseMeta):
     docs_url: Annotated[AnyHttpUrl, StringConstraints(max_length=65536)]
     docs_dev_url: Annotated[AnyHttpUrl, StringConstraints(max_length=65536)]
+    released: dict[str, VersionStr] = Field(
```

Comment:
this was fixed by merging your latest contribution in `master`

Reply:
FYI I tried to use the 
```
docs_dev_url: Annotated[AnyHttpUrl, Field(alias="docsDevUrl")]
```
but then the alias doesn't seem to change the field name in the oas. I am guessing this is the difference between the `pydantic` and `fastapi` schemas that you also saw yourself. Hence, for now I will continue to use the `= Field(alias="docsDevUrl")` notation since I would like to get this PR in before it grows even more.